pkgs/applications/emulators/wine-nine-standalone/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, symlinkJoin, makeWrapper, bash, wine, nine32, nine64 }:
+
+symlinkJoin {
+  inherit (nine64) name pname version meta;
+
+  nativeBuildInputs = [ makeWrapper ];
+  paths = [ wine ];
+
+  postBuild = ''
+    [[ -x "$out"/bin/wine ]] && wrapProgram "$out"/bin/wine \
+      --prefix WINEDLLPATH ':' ${lib.escapeShellArg "${nine32}/lib/wine"} \
+      --prefix WINEDLLPATH ':' ${lib.escapeShellArg "${nine64}/lib/wine"}
+
+    [[ -x "$out"/bin/wine64 ]] && wrapProgram "$out"/bin/wine64 \
+      --prefix WINEDLLPATH ':' ${lib.escapeShellArg "${nine32}/lib/wine"} \
+      --prefix WINEDLLPATH ':' ${lib.escapeShellArg "${nine64}/lib/wine"}
```

Comment:
Isn't this depending on stdenv.is32Bit?

Reply:
How so? The toplevel passes the following inputs to this package:
```
    nine32 = pkgsi686Linux.wine-nine-standalone-unwrapped;
    nine64 = wine-nine-standalone-unwrapped;
```
So on a 32 bit system, nine64 is actually the 32 bit build, and we just prefix WINEDLLPATH with the same value twice (and because of how `wrapProgram --prefix` works, the final value only contains it once).

For pure 32 bit wine packages on a 64 bit system, we do add the 64 bit version to the path, but that's no big deal, wine will just not use it.

Reply:
So we can make ``[[ -x "$out"/bin/wine ]]`` and ``[[ -x "$out"/bin/wine64 ]]`` dependend on stdenv.is32Bit and stdenv.is64Bit?

Reply:
The intent here is to do the right thing whether the input wine is 32 bit, wine64, or WoW.